medusa/tv/series.py

```diff
@@ -335,6 +335,10 @@ def flush_episodes(self):
                 self.episodes[cur_season][cur_ep] = None
                 del my_ep
 
+    def erase_cached_parser(self):
+        """Erase parsed results because user changed xem numbering."""
```

Comment:
I don't think this log message is correct anymore. As we're erasing it for multiple reasons now right?